test/integration/messagesService_integration_tests.js

```diff
@@ -23,7 +25,7 @@ var sinon = salinity.sinon;
 var mockableObject = salinity.mockableObject;
 
 var env = {
-  httpPort: 21000,
+  httpPort: 21003,
```

Comment:
Is there a reason this changed? I'm guessing it doesn't matter to much for a test.

Reply:
It clashed with a test in another repo.